csharp/src/Util.cs

```diff
@@ -52,7 +52,10 @@ internal static void TryAction(Action f, string? message = null)
         internal SignatureException(string message, Exception? innerException) : base(message, innerException) { }
     }
 
-    internal static class Util
+    /// <summary>
+    /// Signing utility class.
+    /// </summary>
+    public static class Util
```

Comment:
If you did this just to use the `ParesePem` method in the tests, please revert it and simply allow the internal objects to be visible to the test project (see [here](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=net-6.0)). In this way, we will avoid exposing internal utility classes and methods to the consumers of the library.

Reply:
> To make things easier, we may want to turn the [ParsePem](https://github.com/TrueLayer/truelayer-signing/pull/83/files#diff-d67f01d5f39f15a2a9c2df8bb129017525d301065eeb9c4258264931e66b451bR89) method as public (if we don't want to make the whole Util class public, we can think of moving the ParsePem to a separate class).

The goal was to expose the `ParsePem` method to the end user so that it could be easier for them to leverage the `Signer.SignWith(<NewGuid>, <EcdsaKey>)`. Having it available in the test project is more of a "side-effect".